staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -174,6 +184,26 @@ func (a *mutatingDispatcher) callAttrMutatingHook(ctx context.Context, invocatio
 		return apierrors.NewInternalError(err)
 	}
 
+	annotationKey := fmt.Sprintf("%s-%d", h.Name, round)
+	mutation := "false"
+	// NOTE: here we determine if a webhook mutated the object by comparing byte to byte.
+	// In future if we find this causing too much false positive, we can switch to use
+	// !apiequality.Semantic.DeepEqual(attr.VersionedObject, newVersionedObject)
+	if !bytes.Equal(objJS, patchedJS) {
+		mutation = "true"
+		if err := attr.Attributes.AddAnnotation(patchAuditAnnotationPrefix+annotationKey, string(patchJS)); err != nil {
+			klog.Warningf("failed to set patch annotation for mutating webhook %s", annotationKey)
```

Comment:
I'm trying to avoid logging the actual patch in apiserver log, to prevent potentially leaking PII from here

cc @liggitt